README.md

```diff
@@ -224,6 +224,21 @@ assert("a" in #{ a: 1, b: 2 });
 
 See the [appendix](./NS-Proto-Appendix.md) to learn more about the `Record` & `Tuple` namespaces.
 
+## `Record` and `Tuple` Wrapper Objects
+
+- `Object(record)` creates an instance of `Record`, which is the wrapper object for `record` values.
+- `Object(tuple)` creates an instance of `Tuple`, which is the wrapper object for `tuple` values.
+
+The `Record` prototype object is an ordinary object, is not a `Record` instance, and its prototype is `Object`.
```

Comment:
Probably you mean `Object.prototype` (and ditto below). Maybe start with "`Record.prototype` is..." for clarity.